src/react/src/components/Contribute/SearchByOsIdSuccessResult.jsx

```diff
@@ -36,7 +39,11 @@ const SearchByOsIdSuccessResult = ({
                 defaultButtonLabel="No, search by name and address"
                 defaultButtonAction={handleBackToSearchByNameAddress}
                 secondaryButtonLabel="Yes, add data and claim"
-                secondaryButtonAction={() => {}}
+                secondaryButtonAction={() => {
+                    history.push(
+                        `/contribute/production-location/${osId}/info/`,
+                    );
+                }}
```

Comment:
Please create a function that handles routing.
secondaryButtonAction={handle...}

Reply:
I've applied separate functions.